index.markdown

```diff
@@ -387,8 +386,7 @@ redirect_from: '/moveit/'
                         <p><span>Version:</span> 2.0</p>
                         <p><span>Platforms:</span> Ubuntu 18.04</p>
                         <p><span>Initial release:</span> 1 June 2019</p>
-                        <p><span>End of life:</span> May 2021</p>
-                        <p><span>Branch: <a href="https://github.com/AcutronicRobotics/moveit2" target="_blank">AcutronicRobotics/moveit2</a></span> </p>
```

Comment:
Why remove this? It is possible someone will find it useful, even if we aren't supporting it

Reply:
reverted (thought this might be outdated by now)